src/engine/CountAvailablePredicates.cpp

```diff
@@ -108,6 +108,8 @@ size_t CountAvailablePredicates::getCostEstimate() {
 
 // _____________________________________________________________________________
 void CountAvailablePredicates::computeResult(ResultTable* result) const {
+  LOG(DEBUG) << "Beginning with computing the CountAvailablePredicates result."
```

Comment:
"Begin computing …" sounds a bit weird to me. I think what that's because "-ing" words are normally for stuff that that takes time and it clashes with "the beginning". Also in "Finished with computing the " one can drop the "with". Both of these are used a bunch of times.

Reply:
I've changed the messages to have the same structure as the messages of older operations (e.g. OrderBy)